docs/guides/admin/docs/configuration/security.jwt.md

```diff
@@ -1,18 +1,107 @@
+
 In order to integrate Opencast with an OIDC provider you could use the
 [oauth2-proxy](https://github.com/oauth2-proxy/oauth2-proxy).
 
+Standard OC Schema for JWTs
+---------------------------
+
+After the signature verification, a JWT is just a bunch of "claims" (JSON fields). How to interpret these claims is completely up to the communicating parties. Protocols like OIDC define exactly which claims are to be used and how. For other cases where external Opencast apps (like Tobira or an LMS) need to communicate with Opencast, this section defines a standard schema to follow. You can configure Opencast differently, but this aims at standardizing JWT usage across Opencast-related applications and offer an out-of-the-box solution.
+
+This schema is designed to be flexible and support all current and conceivable future use cases, while keeping the JWT size for each use case small.
+
+### Quick overview
+
+```json5
+{
+    // Standard claims
+    "exp": 1499863217, // required
+    "nbf": 1548632170, // optional
+
+    // User info
+    "username": "jose",
+    "name": "José Carreño Quiñones",
+    "email": "jose@example.com",
+
+    // Authorization, i.e. privileges the request/user should have
+    "roles": ["STUDIO", "API_EVENTS_VIEW"], // without `ROLE_` prefix
+    "oc": [
+        "read+customaction:e:d622b861-4264-4947-8db1-c754c5956433",
+        "write:s:4ed02421-144c-42a1-b98a-22e84f3ac691"
+    ]
+}
+```
+
+### Claims
+
+A claim being "required" means that the JWT provider (e.g. Tobira, LMS) has to include it. Opencast will reject all JWTs which lack required claims.
+
+- `exp` (expiration time): As [defined in the JWT RFC](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4), a number timestamp in seconds since the unix epoch. JWTs with `exp < now()` are rejected by Opencast. This claim is *required*.
+
+- `nbf` (not before): As [defined in the JWT RFC](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.5), a number timestamp in seconds since the unix epoch. JWTs with `nbf > now()` are rejected by Opencast. This claim is optional.
+
+- *User Info*: information about the user that's possibly used by parts of Opencast. Some JWT use cases don't require user information at all (e.g. static file auth), while others do (e.g. Studio). Therefore, these claims are optional. (TODO: specify when this is stored as user reference)
+    - `username`: string. (TODO: explain how this is used to lookup existing users, if at all)
```

Comment:
This should happen with the usual user/role providers (if we use the JWT for authn).